custom/shared/components/HairCheck/HairCheck.js

```diff
@@ -127,6 +131,39 @@ export const HairCheck = () => {
     }
   }, [camError]);
 
+  const showWaitingMessage = useMemo(() => {
+    return (
+      <div className="waiting">
+        <Loader />
+        {denied ? (
+          <span>Call owner denied request</span>
+        ) : (
+          <span>Waiting for host to grant access</span>
```

Comment:
Was the loader always offset to the left like this? `margin:auto;` on the Loader will center it horizontally. (Nitpick, doesn't really matter but would look better to center.)
<img width="734" alt="CleanShot 2022-01-05 at 16 42 51@2x" src="https://user-images.githubusercontent.com/18265845/148293941-796f9987-1579-4f21-a0be-a18ae6327376.png">

Reply:
@jessmitch42, this was a fun one! 

- The original <Loader /> seems to have been to the left of the "Waiting..." text. The "centered" prop used to center the component wasn't used. 
- But! Memoization did seem to mess up the original inline styling; thanks for catching. I moved the relevant styled jsx up into the memo, because otherwise my CSS wasn't showing up at all. 
- I added a `flex-direction: column` to the main "waiting" div to avoid having to add a new class on the component or style the svg element. 

It all looks centered to me now, but if you'd prefer I fix another way (or if I really shouldn't have had to do that with styled jsx) let me know!